ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -243,3 +243,10 @@ SENSOR_CO2M_VOCM:
     - /SENSOR
     - CO2M
     - VOCM
+
+SENSOR_MAZTM:
```

Comment:
It is not making sense to me to have an abstract type that contains setpoints on a sensor entity. This seems like just a zone temperature sensor to me, and the different setpoints should go on the AHU or whatever this sensor applies to.

Reply:
Okay yeah this probably makes more sense